app/src/functions/notify-endpoint-usage/app.ts

```diff
@@ -31,14 +35,16 @@ export async function getUsageData(): Promise<GetUsageDataQuery[]> {
 function getRelaysUsed(networkData: QueryAppResponse[], influxData: GetUsageDataQuery[]): ApplicationData[] {
   const applicationsData: ApplicationData[] = []
 
+  const influxDataMap: { [any: string]: GetUsageDataQuery } = influxData.reduce((acc, data) => {
```

Comment:
So, I don't really like the naming of this variable because it has the data structure in it haha. We could also use a native [`Map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) here.